extensions/notebook/src/common/utils.ts

```diff
@@ -475,16 +475,17 @@ function hasWorkspaceFolders(): boolean {
 	return workspaceFolders && workspaceFolders.length > 0;
 }
 
-export async function setPinnedBookPathsInConfig(pinnedNotebookPaths: IBookNotebook[]): Promise<void> {
+export async function setPinnedBookPathsInConfig(pinnedNotebookPaths: INotebook[]): Promise<void> {
 	let config: vscode.WorkspaceConfiguration = vscode.workspace.getConfiguration(notebookConfigKey);
 	let storeInWorspace: boolean = hasWorkspaceFolders();
 
 	await config.update(pinnedBooksConfigKey, pinnedNotebookPaths, storeInWorspace ? false : vscode.ConfigurationTarget.Global);
 }
 
 
-export interface IBookNotebook {
+export interface INotebook {
```

Comment:
INotebook is very generic, if this is specific to pinned notebooks, would suggest to name it as INotebookPin